### DISCUSSION TITLE
Chỉnh sửa lại bài báo cáo

### DISCUSSION THREAD

**Discussion by dangxuantruong10a13** [dangxuantruong10a13/nhom9_10a13#15] [Announcements]
Theo như thầy nhận xét, bài của nhóm mình lung tung và cần trang trí. Nhưng mà lung tung thì mình không biết còn trang trí thì chuẩn đúng rồi. Vậy nên bây giờ mình chia task cho các bạn làm trong 2 tiếng (Gất dễ). @TranBaoLam10C13 làm lại bố cục của bài sao cho vjp vjp xì líp, @NgHan12pn tìm ảnh để trang trí cho bài, @04lananh tìm lại những lỗi sai của bài. Sau khi làm xong đưa cho mình để mình tạo thành một file word, sau đó các bạn tìm lỗi sai rồi nói mình nhé

**Comment by NgHan12pn** [comment]
aloo tui up lên đây nhen

**Comment by 04lananh** [comment]
tui là tui thấy bài oke rùi nha
trình bày đẹp quá trời luôn

**Reply by dangxuantruong10a13** [reply]
Oke xịn xịn

**Reply by dangxuantruong10a13** [reply]
Nhưng mà thầy nói lung tung, không đepn cần trang trí lại á nên phải trang trí nèeee

**Reply by 04lananh** [reply]
sao tiêu chuẩn thầy cao thế

**Comment by NgHan12pn** [comment]
![image](https://user-images.githubusercontent.com/91815517/139284725-5c850cd7-39cf-400f-83eb-8e3d4df6f364.png)
![image](https://user-images.githubusercontent.com/91815517/139284840-92ffc037-c77a-46cd-b614-ac9040dc7db9.png)
![image](https://user-images.githubusercontent.com/91815517/139284952-d8cae15b-a854-4410-b8a9-6e491f91033d.png)
![image](https://user-images.githubusercontent.com/91815517/139285017-ee60fad6-c523-4be3-8aef-307c071add0b.png)

**Comment by NgHan12pn** [comment]
Đợi xíu tìm tiếp r up lên nhaaa

**Comment by NgHan12pn** [comment]
![image](https://user-images.githubusercontent.com/91815517/139285291-020550df-284b-4da0-bd7c-2b1a8ff26009.png)
![image](https://user-images.githubusercontent.com/91815517/139285363-a86781b8-a396-4829-a817-3d9fdbefe641.png)
![image](https://user-images.githubusercontent.com/91815517/139285479-dd3fda5c-e7ba-41e1-8c94-23b4ef0fe5b7.png)
![image](https://user-images.githubusercontent.com/91815517/139285593-1c340bcb-14a5-48a8-8d83-485e1fe1ca8f.png)
![image](https://user-images.githubusercontent.com/91815517/139285662-20da9a25-45af-4996-874c-018ef66bc628.png)
![image](https://user-images.githubusercontent.com/91815517/139285740-435ef302-b1fb-4111-9fd8-9a351626a1c6.png)

**Comment by NgHan12pn** [comment]
![image](https://user-images.githubusercontent.com/91815517/139285938-2286f50d-4a5a-45be-ac0c-884a70e7b040.png)
![image](https://user-images.githubusercontent.com/91815517/139285995-417f7a65-ed3d-4b87-924f-77e4675a80f7.png)
![image](https://user-images.githubusercontent.com/91815517/139286041-08fad18e-cdf2-4774-b8f1-344e08a71247.png)
![image](https://user-images.githubusercontent.com/91815517/139286109-b3c27857-c0ac-4c5b-9dab-7ec50a376d42.png)
![image](https://user-images.githubusercontent.com/91815517/139286146-1e19103e-4f71-4a8b-9a8b-19dbd2de080f.png)
![image](https://user-images.githubusercontent.com/91815517/139286229-75d5516f-e901-4b6a-ad3f-c346ae2c69f3.png)
![image](https://user-images.githubusercontent.com/91815517/139286282-877d4dd6-8a91-4cdd-b567-863f0fb535b6.png)
![image](https://user-images.githubusercontent.com/91815517/139286375-0e497515-3519-4430-82e5-bae4c0d5cd30.png)

**Comment by NgHan12pn** [comment]
hết ý tưởng tìm tiếp gòyy

**Comment by NgHan12pn** [comment]
kiếm được nhiêu đó thôi á

**Reply by dangxuantruong10a13** [reply]
Oke cám ơn nhaa

**Comment by dangxuantruong10a13** [comment]
@TranBaoLam10C13 rầu giờ ông lấy ảnh phía trên Hân gửi r sắp xếp bố cục nhé

**Reply by TranBaoLam10C13** [reply]
oke

**Comment by TranBaoLam10C13** [comment]
![Untitled1](https://user-images.githubusercontent.com/91810620/139303972-606b5fed-a7de-430d-8495-a3cae76464ca.png)
![Untitled](https://user-images.githubusercontent.com/91810620/139303979-aa571b03-5476-4d77-95de-e3fd602b5475.png)

**Reply by TranBaoLam10C13** [reply]
Cái này là bố cục mẫu

**Reply by dangxuantruong10a13** [reply]
oke tui lụm bỏ vô bài rồi
bài tui mới up ở dưới nè

**Comment by dangxuantruong10a13** [comment]
[1 - Nhóm 9 - Báo cáo bài 4 Tin Học (beta).pdf](https://github.com/dangxuantruong10a13/nhom9_10a13/files/7446228/1.-.Nhom.9.-.Bao.cao.bai.4.Tin.H.c.beta.pdf)

@04lananh 
@TranBaoLam10C13 
@NgHan12pn

Mọi người vào xem file có gì thiếu hay lỗi chính tả, sửa chữa gì không? Mình sửa trong hôm nay nha, sắp nộp cho thầy vào ngày mai rồi

**Reply by TranBaoLam10C13** [reply]
Ông ơi thiếu Chủ đề con: BÀI TOÁN VÀ THUẬT TOÁN

**Reply by dangxuantruong10a13** [reply]
omg để sửa

**Reply by NgHan12pn** [reply]
Ủa hình như câu 7 thiếu câu hỏi kìa

**Reply by dangxuantruong10a13** [reply]
o,o oke chắc tui buồn ngủ lúc copy paste :'

**Reply by dangxuantruong10a13** [reply]
chắc hết rồi đk mn?

**Reply by 04lananh** [reply]
nguồn của câu 5 nè : https://loigiaihay.com/cau-5-trang-44-sgk-tin-hoc-10-c156a24885.html?fbclid=IwAR1KoEs3oPYm-OxqwSS9IcldB3fy9S-dY5N_0faVzo9QGxE3KbpsiKEQW3U

**Reply by dangxuantruong10a13** [reply]
oke tui thêm vào hết r đó. Còn gì nữa khum nhỉ

**Comment by dangxuantruong10a13** [comment]
[1 - Nhóm 9 - Báo cáo bài 4 Tin Học.pdf](https://github.com/dangxuantruong10a13/nhom9_10a13/files/7447118/1.-.Nhom.9.-.Bao.cao.bai.4.Tin.H.c.pdf)

Okay bản này sẽ gửi cho thầy vào ngày mai
